From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chara <bionichuman07@gmail.com>
Date: Mon, 27 Apr 2026 00:46:07 -0400
Subject: [PATCH 30/31] [RestlessOS PATCH] Replace Google NTP server with
 pool.ntp.org

The default NTP server time.android.com is a Google-operated endpoint.
Replace it with pool.ntp.org which is the community-maintained NTP Pool
Project, removing an unnecessary Google network dependency.

Attribution for this commit goes to chrisaw (<contact@chrisaw.io>)
---
 core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index e05f44db15..487bb3b416 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -2780,7 +2780,7 @@
          Items must be in the form: "ntp://<host>[:port]"
          This is not a registered IANA URI scheme. -->
     <string-array translatable="false" name="config_ntpServers">
-        <item>ntp://time.android.com</item>
+        <item>ntp://pool.ntp.org</item>
     </string-array>
     <!-- SNTP client config: Timeout to wait for an NTP server response in milliseconds. -->
     <integer name="config_ntpTimeout">5000</integer>
